src/containers/TimeseriesPreview.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Modal } from 'antd';
+import { TimeseriesChartMeta } from '@cognite/gearbox';
+import { Dispatch, bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { selectTimeseries, TimeseriesState } from '../modules/timeseries';
+import { RootState } from '../reducers/index';
+import { selectApp, AppState, setTimeseriesId } from '../modules/app';
+
+type Props = {
+  timeseries: TimeseriesState;
+  app: AppState;
+  setTimeseriesId: typeof setTimeseriesId;
+};
+
+class TimeseriesPreview extends React.PureComponent<Props, {}> {
+  render() {
+    const {
+      timeseries: { timeseriesData },
+      app: { timeseriesId },
+    } = this.props;
+    let timeseries;
+    if (timeseriesId) {
+      timeseries = timeseriesData[timeseriesId];
+    }
+    return (
+      <Modal
+        visible={timeseriesId !== undefined}
+        width={1000}
```

Comment:
Should we have hard coded widths?

Reply:
Eventually these modules will become components, I do not like how everything is a modal atm.